src/xmpp/presence.c

```diff
@@ -656,6 +658,69 @@ _available_handler(xmpp_stanza_t* const stanza)
 
     if (g_strcmp0(xmpp_presence->jid->barejid, my_jid->barejid) == 0) {
         connection_add_available_resource(resource);
+        char* account_name = session_get_account_name();
+        int max_sessions = accounts_get_max_sessions(account_name);
+        char* cur_resource = accounts_get_resource(account_name);
+        if (max_sessions > 0) {
+            int res_count = connection_count_available_resources();
+            if (res_count > max_sessions && g_strcmp0(cur_resource, resource->name)) {
+                ProfWin* console = wins_get_console();
+                ProfWin* current_window = wins_get_current();
+                auto_gchar gchar* message = g_strdup_printf("Max sessions alarm! (%d/%d devices in use)", res_count, max_sessions);
+                win_println(console, THEME_RED, "|", message);
+                if (console != current_window) {
+                    win_println(current_window, THEME_RED, "|", message);
```

Comment:
That's the only place where we print in the `current_window`.
Would it maybe make sense to either add some explanation that the console window has more information? Or print everything in both windows? IMO adding an explanation here should be sufficient.
```suggestion
                    win_println(current_window, THEME_RED, "|", "%s - check the console for more details!", message);
```

Reply:
Hmm yes actually the statusbar indicates a change in the console window if something is printed there and the user should check there. I even wouldn't print the notice about console. Since it's like this in all (?) commands.